src/types/index.tsx

```diff
@@ -42,6 +42,36 @@ export type UserDetails = {
   status: UserStatus;
 };
 
+export interface LeaveRequestData {
```

Comment:
Not a fan of interfaces with word `Data` in them.

Reply:
I have fixed this by renaming as `LeaveRequestDetails`